src/auto-evo/RunResults.cs

```diff
@@ -782,11 +807,37 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                     }
                 }
 
-                if (GetGlobalPopulation(entry.Species, resolveMigrations, resolveSplits) <= 0)
+                var globalPopulation = GetGlobalPopulation(entry.Species, resolveMigrations, resolveSplits);
+                var previousTotalPopulations = previousPopulations.GetSpeciesPopulation(entry.Species);
+
+                if (globalPopulation <= 0)
                 {
                     builder.Append(' ');
                     builder.Append(new LocalizedString("WENT_EXTINCT_FROM_PLANET"));
                     builder.Append('\n');
+
+                    world.LogWorldEvent(new LocalizedString("TIMELINE_SPECIES_EXTINCT",
```

Comment:
This is now also vulnerable to this issue: https://github.com/Revolutionary-Games/Thrive/issues/2958

Reply:
Added TODO.